platform/client-admin/.graphqlrc.yml

```diff
@@ -15,6 +15,9 @@ extensions:
       subUpdate: false
       subDelete: false
     plugins:
+      graphback-schema:
+        format: graphql
+        outputPath: ../server/src/schema
```

Comment:
See this comment https://github.com/aerogear/OpenVolunteerPlatform/pull/56/files#r421332884